### DISCUSSION TITLE
Markdown格式下行内代码如何设置颜色

### DISCUSSION THREAD

**Discussion by lbx-aike** [vuepress-theme-hope/vuepress-theme-hope#2782] [Q&A] [answered]
Markdown格式下行内代码如何设置颜色
```markdown
`改变字体颜色`
```
`改变字体颜色`
如何变成下面这样
![image](https://user-images.githubusercontent.com/20965566/218698452-b5ad957a-1874-42f1-b763-d7263cdb6e6a.png)

**Comment by Mister-Hope** [comment, marked as answer]
https://developer.mozilla.org/zh-CN/docs/Web/CSS
